src/components/tools/text-tool.tsx

```diff
@@ -241,4 +272,99 @@ export default class TextToolComponent extends BaseComponent<IProps, IState> {
     return this.props.model.content as TextContentModelType;
   }
 
+  private makeOnKeyDownHandler(hotKeyDef: IOnKeyDownHandlerDef): Plugin {
```

Comment:
As we discussed, we should refactor this handler-builder after we have some unit tests in place. But for now, this should be good enough.